sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java

```diff
@@ -75,6 +69,31 @@
       }
     } else if (id == R.id.action_web_map) {
       airMapInterface = mapViewBuilder.builder(AirMapViewTypes.WEB).build();
+    } else if (id == R.id.action_web_map_no_touch) {
+      airMapInterface = mapViewBuilder.builder(AirMapViewTypes.WEB).build();
+      final AirMapInterface finalAirMapInterface = airMapInterface;
+      airMapInterface.setOnMapLoadedListener(new OnMapLoadedListener() {
+        @Override
+        public void onMapLoaded() {
+          finalAirMapInterface.setTouchEnabled(false);
+        }
+      });
+    } else if (id == R.id.action_native_map_no_touch) {
+      try {
+        airMapInterface = mapViewBuilder.builder(AirMapViewTypes.NATIVE).build();
+        final AirMapInterface finalAirMapInterface1 = airMapInterface;
+        airMapInterface.setOnMapLoadedListener(new OnMapLoadedListener() {
+          @Override
+          public void onMapLoaded() {
+            finalAirMapInterface1.setTouchEnabled(false);
+          }
+        });
```

Comment:
Ideally we'd allow for full control of all of of these different methods - if you could add support for them that'd be cool, otherwise I'm fine with using `setAllGuesturesEnabled` for now, and I can add support for the others.